website/docs/d/datasource_cloudfunctions_function.html.markdown

```diff
@@ -45,11 +45,10 @@ exported:
 * `description` - Description of the function.
 * `available_memory_mb` - Available memory (in MB) to the function.
 * `timeout` - Function execution timeout (in seconds).
+* `runtime` - The runtime in which the function is running.
 * `entry_point` - Name of a JavaScript function that will be executed when the Google Cloud Function is triggered.
 * `trigger_http` - If function is triggered by HTTP, this boolean is set.
 * `event_trigger` - A source that fires events in response to a condition in another service. Structure is documented below.
-* `trigger_bucket` - If function is triggered by bucket, bucket name is set here. Deprecated. Use `event_trigger` instead.
```

Comment:
These are still deprecated as of this PR, so if it's not too much effort, maybe keep them in the docs (but nbd if it would be painful)